### PR TITLE
[refs #00134] Add sitemap template

### PR DESCRIPTION
**Pull request by Android63**
Page template for displaying sitemap.

### CODE REVIEW

File: page-sitemap.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package nightingale-wp
+ */
+
+get_header(); ?>
+<div class="o-layout  o-layout--wide">
+
+	<div id="primary" class="o-layout__item u-8/12@lg">
```
Comment (cehwitham) [inline page-sitemap.php]: Should this be full width?

File: page-sitemap.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+		<main id="main" class="site-main" role="main">
+
+			<h2 id="pages">Pages</h2>
+						<ul>
```
Comment (cehwitham) [inline page-sitemap.php]: Indentation seems a bit off? Can we tidy this up please?

File: page-sitemap.php (hunk 1 above)
Comment (Android63) [reply page-sitemap.php]: Yes. I've fixed it.

File: page-sitemap.php (hunk 2 above)
Comment (Android63) [reply page-sitemap.php]: Done.

**Comment by Android63** [comment]
@cehwitham could you review again please?

**Pull request merged**